.circleci/images/primary/Dockerfile-jruby-9.2.21.0

```diff
@@ -1,24 +1,11 @@
-# NOTE BEFORE YOU UPGRADE TO A LATER JRUBY 9.2 VERSION:
-# There's an incompatibility in JRuby 9.2.19.0+ when run on Java 8 (jre8) with the ethon gem, see
-# https://github.com/jruby/jruby/issues/7033 .
-# The issue goes away if we run on Java 11, but we probably want to test on 8.
-# So, as of this writing (January 2022) we cannot have it all: latest ethon, ethon testing enable, latest JRuby 9.2,
-# and Java 8. One always has to give.
-#
-# since next one is 9.2.19.0 at https://github.com/jruby/docker-jruby/blob/d083a9c13640eba8c135caeefbff3b89350bbede/9.2/jre8/Dockerfile
-# we took the following from this one which is 9.2.11 as default but AFAICT was used to build 9.2.18
-# https://github.com/jruby/docker-jruby/blob/f94681bb26c3da5159e8c0ba9842005a011a76de/9000/jre8/Dockerfile
-#
-# a more recent one would be https://github.com/jruby/docker-jruby/blob/9881bee2d7510aa17dba6b1d7ae5ff711fddd624/9.2/jre8/Dockerfile
-#
-# openjdk:8-jre image is from https://github.com/docker-library/openjdk/blob/a0c4da867ddd1a11408b7ec5032f12130bdc5476/8/jre/bullseye/Dockerfile
+# openjdk:8-jre image is from https://github.com/docker-library/openjdk/blob/a0c4da867ddd1a11408b7ec5032f12130bdc5476/8/jre/buster/Dockerfile
 # note: docker-library/openjdk is deprecated, there is a later move to https://hub.docker.com/_/eclipse-temurin
-FROM openjdk:8-jre-buster AS jruby-9.2.18.0-jre8
+FROM openjdk:8-jre-buster AS jruby-9.2.21.0-jre8
```

Comment:
I strongly think it's important to leave here on and the 9.3 Dockerfile a note about the potential issue with ethon + Debian 11/bullseye.

Otherwise, maybe in a year or two a future colleague will run into the issue again and potentially lose a lot of time isolating it again.

Reply:
Good point!

Reply:
Done!